features/benchmarking.feature

```diff
@@ -1,6 +1,15 @@
 Feature: Benchmarking
   Tests to check the loading times for various pages on GOV.UK.
 
+  # TODO: IMPROVE this test.
+  #
```

Comment:
Improve or remove? If you want to drop the timing then it no longer meets the stated reason for being there in the feature description.

Reply:
It's a bit of both really: the performance steps don't meet the eligibility criteria, but the rest of the scenario is OK and is the only scenario we have that's testing the Bouncer app, so we should try to keep it.

Unlike all the other scenarios in here, which are covered elsewhere, my intention is to just remove the offending steps and move / rename the scenario so its value is clear - as the sole test Bouncer is working.

Does that help? I can tweak the comment but want to check I understand first.

Reply:
Yeah, that makes sense.